src/components/Loader.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { CircularProgress } from '@material-ui/core';
+
+const Loader = ({ size }) => {
+  return (
+    <div className="loading">
+      <CircularProgress size={size} />
+    </div>
+  );
+};
```

Comment:
since there is no logic in the function above the `return`, you could remove the `return` keyword and do an implicit `return`:

```jsx
const Loader = ({ size }) => (
  <div className="loading">
    <CircularProgress size={size} />
  </div>
);
```